openforcefield/data/test_forcefields/GBSA_HCT-1.0.offxml

```diff
@@ -0,0 +1,18 @@
+<SMIRNOFF version="0.3" aromaticity_model="OEAroModel_MDL">
+    <GBSA version="0.3" gb_model="HCT" solvent_dielectric="78.5" solute_dielectric="1" sa_model="ACE" surface_area_penalty="5.4*calories/mole/angstroms**2" solvent_radius="1.4*angstroms">
```

Comment:
Can we add some provenance information for each of these experimental `.offxml` files? Where does this come from?

Reply:
Added provenance info in 93f9c93. The GBSA parameters in the new files are adapted from the contents of [this file](https://github.com/openforcefield/openforcefield/blob/master/openforcefield/data/test_forcefields/Frosst_AlkEthOH_GBSA.offxml#L11) , which states that you originally wrote these terms. I subsequently changed the numbers to fit the the new spec and proper radius/screen values, so I've listed both of us as authors for the FFs.